plugin.yaml

```diff
@@ -160,6 +160,25 @@ node_types:
                     Defaults to GMT without daylight savings.
                 type: integer
                 default: 90
+            windows_organization:
+                description: >
+                    Organization name to use for sysprep customization.
+                type: string
+                default: Organization
+            use_custom_sysprep:
```

Comment:
I considered that, but our approach in the past has tended towards including the 'use_<thing>' parameters as well (e.g. see rabbit ssl where my initial intent was in line with what you suggest here, but was changed after code review).

I was also considering that a user might wish to simply not use the default sysprep approach, and perform no actions other than generalising, which I think they might be able to do by providing an empty string for the use_custom_sysprep... but I'm not 100% sure that'll behave in a useful way (it won't error, but I'm not sure what it will do). Still, this'd only be a useful feature in that case if a single blueprint that could do either was wanted.

So... we can go either way, though I think removing the extra variable is probably better?